custom-connectors/Staffbase/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2691 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Staffbase",
+    "description": "This connector provides access to your Staffbase instance",
+    "version": "1.0",
+    "contact": {
+      "name": "Staffbase Support",
+      "url": "https://www.staffbase.com",
+      "email": "support@staffbase.com"
+    }
+  },
+  "host": "powerautomateproxy-main-de1.staffbase.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/channels": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "total"
+                },
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "limit"
+                },
+                "offset": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "offset"
+                },
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "config": {
+                        "type": "object",
+                        "properties": {
+                          "localization": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "title": {
+                                  "type": "string",
+                                  "description": "title"
+                                },
+                                "locale": {
+                                  "type": "string",
+                                  "description": "locale"
+                                }
+                              }
+                            },
+                            "description": "localization"
+                          }
+                        },
+                        "description": "config"
+                      },
+                      "spaceID": {
+                        "type": "string",
+                        "description": "spaceID"
```

Comment:
@Nm-jens Please provide valid description values for the properties as these are looking like IDs

Documentation - [Description values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the invalid description values in the swagger